### DISCUSSION TITLE
[New discussion] button missing when a new user first come to discussion

### DISCUSSION THREAD

**Discussion by makerjackie** [community/community#2991] [Discussions] [answered]
I suppose it should be there, once the new user click the `new discussion` button, it will suggest the user to sign up github first

**Comment by becca** [comment, marked as answer]
Duplicate of https://github.com/github/feedback/discussions/29. This is on our radar, thank you.

**Reply by makerjackie** [reply]
@becca it is different, I mean, a new user is who haven't sign up for github yet

also, I check my discussion setting:
![image](https://user-images.githubusercontent.com/18050469/102102577-f270d800-3e66-11eb-8493-62e5823c263f.png)

**Reply by becca** [reply]
Both scenarios don't provide any clear visual indicator to the end user as to why discussion posting is not available - we plan to resolve that.

**Reply by makerjackie** [reply]
I just find out I was missing this [comment](https://github.com/github/feedback/discussions/29#discussioncomment-139055), sorry, this discussion is duplicated
